x/ecocredit/core/msg_create_batch_test.go

```diff
@@ -232,7 +231,7 @@ func TestMsgCreateBatch(t *testing.T) {
 				ProjectId: "C01",
 				StartDate: &startDate,
 				EndDate:   &endDate,
-				Metadata:  simtypes.RandStringOfLength(r, 288),
+				Metadata:  string(make([]byte, 288)),
```

Comment:
we could create a a random string at the package level, and reuse it in tests.

Reply:
Not necessary here when all we are checking is if the metadata is too long.